From 8a1e6d19631b1e99947e7e81e00920ac80c08b7b Mon Sep 17 00:00:00 2001
From: Jack Zhu <jack.zhu@starfivetech.com>
Date: Wed, 25 Oct 2023 11:14:22 +0800
Subject: [PATCH 9/9] media: staging: media: starfive: camss: Add TODO file

Add a TODO.txt file to explain what needs to be done to get them
out of staging.

Signed-off-by: Jack Zhu <jack.zhu@starfivetech.com>
---
 drivers/staging/media/starfive/camss/TODO.txt | 4 ++++
 1 file changed, 4 insertions(+)
 create mode 100644 drivers/staging/media/starfive/camss/TODO.txt

diff --git a/drivers/staging/media/starfive/camss/TODO.txt b/drivers/staging/media/starfive/camss/TODO.txt
new file mode 100644
index 000000000..7d459f4f0
--- /dev/null
+++ b/drivers/staging/media/starfive/camss/TODO.txt
@@ -0,0 +1,4 @@
+Unstaging requirements:
+- Add userspace support which demonstrates the ability to receive statistics and
+  adapt hardware modules configuration accordingly;
+- Add documentation for description of the statistics data structures;
-- 
2.40.1

